### PR TITLE
Bump harvester-cloud-provider to v0.2.4

### PR DESCRIPTION
**Pull request by FrankYang0529**
https://github.com/harvester/harvester/issues/4947

### CODE REVIEW

**Review by bk201 — APPROVED** [review]

**Comment by bk201** [comment]
@FrankYang0529 I suggest we include this change too: https://github.com/harvester/charts/pull/229

**Comment by FrankYang0529** [comment]
> @FrankYang0529 I suggest we include this change too: #229

Sounds good. Let's wait for #229 and backport it to this PR.

File: charts/harvester-cloud-provider/Chart.yaml [hunk 1]
```diff
@@ -31,7 +31,7 @@ annotations:
   catalog.cattle.io/kube-version: '>= 1.23.0-0 < 1.27.0-0'
```
Comment (w13915984028) [inline charts/harvester-cloud-provider/Chart.yaml:31]: kube-version be `< 1.28.0-0`

File: charts/harvester-cloud-provider/Chart.yaml [hunk 2]
```diff
@@ -31,7 +31,7 @@ annotations:
...
   catalog.cattle.io/release-name: harvester-cloud-provider
   catalog.cattle.io/os: linux
-  catalog.cattle.io/rancher-version: '>= 2.7.0-0 < 2.8.0-0'
+  catalog.cattle.io/rancher-version: '>= 2.7.0-0 < 2.9.0-0'
```
Comment (w13915984028) [inline charts/harvester-cloud-provider/Chart.yaml:34]: please set `< 2.10.0-0`  to support rancher v2.9 as well

**Review by w13915984028 — COMMENTED** [review]
please update those 2 versions, thanks.

File: charts/harvester-cloud-provider/Chart.yaml (hunk 2 above)
Comment (FrankYang0529) [reply charts/harvester-cloud-provider/Chart.yaml]: Updated it. Thanks.

**Review by w13915984028 — APPROVED** [review]
LGTM, thanks.

**Review by bk201 — APPROVED** [review]
lgtm!

**Pull request merged**